### ISSUE TITLE
When headers is null, auth header is removed

### ISSUE THREAD

**Issue by PierBover** [elbywan/wretch#52]
I set the auth header like this which works fine:

```js
api.auth(`Bearer ${ token }`);
```
On some requests I need to pass headers, but not others. I have a generic method for doing POST requests so sometimes `headers` is undefined and it seems the `Authorization` header is removed:

```js
api.headers(headers)
```

I've resorted to doing this:
```js
api.headers(headers || {})
```

It works, but it doesn't seem like `.headers()` should be messing with the auth header set with `.auth()`, no?

**Comment by elbywan** [comment]
Hi @PierBover,

> It works, but it doesn't seem like .headers() should be messing with the auth header set with .auth(), no?

Indeed this is an issue, thanks for the report!

I just released the `1.5.3` which contains the fix 👍

**Comment by PierBover** [comment]
Awesome! Thanks for the quick fix!
